api/v1/section/service.proto

```diff
@@ -1,18 +0,0 @@
-syntax = "proto3";
-package v1.section;
-import "v1/article/article.proto";
-
-option java_multiple_files = true;
-option java_package = "de.stroeer.api.v1.section";
-
-message GetSectionResponse {
-  repeated v1.article.Article article = 1;
```

Comment:
Was wurde aus dem SectionService?

Reply:
separate folder is there, we just need the IDL